compute/bare-metal-cloud/concepts.mdx

```diff
@@ -40,7 +40,7 @@ Bare Metal servers allow you to order dedicated servers on-demand, like cloud In
 
 ## Flexible IP
 
-Flexible IP addresses are public IP addresses which you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
+flexible IP addresses are public IP addresses which you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
```

Comment:
Start of a sentence, I think it should retain its cap

Reply:
```suggestion
Flexible IP addresses are public IP addresses that you can hold independently of any Instance. When you create a Scaleway Instance, its public IP address is also a flexible IP address. Flexible IP addresses can be attached to and detached from any of your Instances as you wish. You can keep a number of flexible IP addresses in your account at any given time. When you delete a flexible IP address, it is disassociated from your account to be used by other users.
```